.github/workflows/node-ci.yml

```diff
@@ -72,41 +80,6 @@ jobs:
       run: yarn flow:ci
 
 
-  coverage:
```

Comment:
This has been rolled into the publish step, which was already running tests anyway.